test/python/smoketest.py

```diff
@@ -4,4 +4,4 @@
 from torch_mlir.dialects import torch
 
 with torch_mlir.ir.Context() as ctx:
-  torch.register_dialect(ctx)
```

Comment:
Depending on what you do above, can remove this part of the test, but it would be nice to maybe create one of the torch types or something (which would just test that things in fact did register).